runtime/runtime-params-estimator/README.md

```diff
@@ -4,22 +4,40 @@ Use this tool to measure the running time of elementary runtime operations that
 
 1. Run the estimator
     ```bash
-    cargo run --release --package runtime-params-estimator --features required --bin runtime-params-estimator -- --accounts-num 20000 --additional-accounts-num 200000 --iters 1 --warmup-iters 1 --metric time
+    cargo run --release --package runtime-params-estimator --features required \
+        --bin runtime-params-estimator -- --accounts-num 20000 --additional-accounts-num 200000 \
+        --iters 1 --warmup-iters 1 --metric time
     ```
 
-    With the given parameters above estimator will run relatively fast.
-    Note the `--metric time` flag: it instructs the estimator to use wall-clock time for estimation, which is quick, but highly variable between runs and physical machines.
-    To get more robust estimates, use these arguments:
+    With the given parameters above estimator will run relatively fast. Note the `--metric time`
+    flag: it instructs the estimator to use wall-clock time for estimation, which is quick, but
+    highly variable between runs and physical machines.
+
+    To get more robust estimates, replace the arguments that follow `--` with these:
 
     ```bash
     --accounts-num 20000 --additional-accounts-num 200000 --iters 1 --warmup-iters 1 \
       --docker --metric icount
     ```
 
-    This will run and build the estimator inside a docker container, using QEMU to precisely count the number of executed instructions.
+    This will run and build the estimator inside a docker container, using QEMU to precisely count
+    the number of executed instructions.
+
+    Alternatively you can skip docker during the development and iterate as such (actual production
+    estimations should still utilize docker to ensure reproducibility):
+
+    ```
+    cargo build --profile=dev-release -p runtime-params-estimator --features required
+    qemu-x86 -cpu Westmere-v1 -plugin file=./emu-cost/counter_plugin/libcounter.so ../../target/dev-release/runtime-params-estimator $ARGS
```

Comment:
Since everybody is using a different system, the instructions on how to do that are going to be different. Besides the only package manager I can reasonably write any documentation for is Nix. I would imagine a developer will know how to do it, not to mention that currently there's a pre-compiled binary copy in the repository as well.